semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -123,7 +123,10 @@ def __init__(self, group):
 
         self.semconv_id = self.id
         self.note = group.get("note", "").strip()
-        self.prefix = group.get("prefix", "").strip()
+        if group.get("prefix", "") is None:
```

Comment:
I don't think this can ever be None? Except if you actually write `null` in the YAML. I.e., I don't understand why this was required. What was the error?

Reply:
Workflow run: https://github.com/open-telemetry/build-tools/runs/7694090819?check_suite_focus=true

The error was:
```
        self.semconv_id = self.id
        self.note = group.get("note", "").strip()
>       self.prefix = group.get("prefix", "").strip()
E       AttributeError: 'NoneType' object has no attribute 'strip'
```

Reply:
Maybe in YAML an empty attribute is equivalent to setting it to `null`? Can you try `prefix: ""` in the YAML?

Reply:
Done.  That works.

Reply:
It's always surprising how surprising a "simple" YAML file can behave...